loader/src/loader.c

```diff
@@ -21,14 +21,26 @@ N.B. playback vs capture is determined by the codec choice
 
 void device_release_callback(struct device *dev) { /* do nothing */ };
 
+#ifdef RPI_4B
+    #define CARD_PLATFORM_STR   "fe203000.i2s"
+#else
+    #define CARD_PLATFORM_STR   "3f203000.i2s"
+#endif
+
+#ifdef I2S_MASTER
+    #define SND_SOC_DAIFMT_CBS_FLAG SND_SOC_DAIFMT_CBS_CFS
+#else
+    #define SND_SOC_DAIFMT_CBS_FLAG SND_SOC_DAIFMT_CBS_CFM
```

Comment:
I am pretty sure this is wrong - it is either:

SND_SOC_DAIFMT_CBS_CFS

or

SND_SOC_DAIFMT_CBM_CFM

There should be no case where the bit clock is slave and frame synch (LR) is master

Reply:
Well spotted! That would have been tricky to debug!